addon/-private/sprite.js

```diff
@@ -461,6 +461,38 @@ export default class Sprite {
     this._finalCumulativeTransform = cumulativeTransform(this.element);
```

Comment:
This is now causing us to calculate the cumulative transform twice. I think this line can just become `this._finalCumulativeTransform = this.cumulativeTransform`, and then it will share the cache.